test/common/README.md

```diff
@@ -935,6 +936,19 @@ listener to process `'beforeExit'`. If a file needs to be left open until
 Node.js completes, use a child process and call `refresh()` only in the
 parent.
 
+## UDP pair helper
```

Comment:
Could be factored out, though additions to the test library are unlikely to raise concerns, but if it relies on src changes, would be better standalone.